### PR TITLE
Kubelet restart

### PR DESCRIPTION
**Pull request by madhusudanupadhyay**
### What type of PR is this?

Managed Script addition for Kubelet restart

### What this PR does / Why we need it?

- Restart Kubelet on Master node. 
-  Customer with cluster admin perms can do it but since the action needs to be on Master node, a preferable way would be doing it via managed script by MCS secondment engineers
- Issues like: pod stuck in containerUnknown Status, bugs that require kubelet restarts to unblock upgrades etc. 

### Which Jira/Github issue(s) does this PR fix?

From one of the use cases, an example could be: https://access.redhat.com/solutions/6976343

### Special notes for your reviewer
Reducing an OHSS and turn around time for customers by letting MCS secondment engineers quickly restart the kubelet in scenarios where required.

### CODE REVIEW

File: scripts/CEE/kubelet-restart/script.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -e
```
Comment (clcollins) [inline scripts/CEE/kubelet-restart/script.sh:3]:
```suggestion
set -o errexit
```

...if you want it to match :D

File: scripts/CEE/kubelet-restart/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Restart Kubelet on Master Node
```
Comment (clcollins) [inline scripts/CEE/kubelet-restart/README.md:1]: How about generic "Restart Kubelet on a Node"

File: scripts/CEE/kubelet-restart/script.sh (hunk 1 above)
Comment (madhusudanupadhyay) [reply scripts/CEE/kubelet-restart/script.sh]: Done! Thanks :)

File: scripts/CEE/kubelet-restart/README.md (hunk 2 above)
Comment (madhusudanupadhyay) [reply scripts/CEE/kubelet-restart/README.md]: Changes made for it to not be node-specific.! Thanks!

File: scripts/CEE/kubelet-restart/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Restart Kubelet on Node
+
+This will script will restart kubelet on Node. Scenarios where a restart might be required:
```
Comment (bmeng) [inline scripts/CEE/kubelet-restart/README.md:3]:
```suggestion
This script will restart kubelet on Node. Scenarios where a restart might be required:
```

File: scripts/CEE/kubelet-restart/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+file: script.sh
+name: restart-kubelet
+description: Restart kubelet for a specific node
+author: Madhusudan Upadhyay
+allowedGroups:
+  - SREP
+  - LPSRE
+  - CEE
+rbac:
+  roles:
```
Comment (bmeng) [inline scripts/CEE/kubelet-restart/metadata.yaml:10]: why do we need both the namespaced role and cluster role here?

**Comment by bmeng** [comment]
@jaybeeunix would you mind to take a look at this script from compliance point of view?

**Comment by jaybeeunix** [comment]
it looks good from my perspective.

**Pull request merged**